src/templates/Manage/CreatePool/ConfigureFee/FeeConfig/index.tsx

```diff
@@ -47,7 +47,7 @@ const FeeConfig = () => {
     dispatch(setFee({ inputName: inputName, inputValue: inputValue }))
   }
 
-  function handlerefferalCommission(
+  function handleRefferalCommission(
```

Comment:
It is spelled "referral"

```suggestion
  function handleReferralCommission(
```